intel_extension_for_transformers/llm/runtime/graph/core/ne_layers.c

```diff
@@ -7867,9 +7874,8 @@ static void ne_compute_forward_rope_f32(const struct ne_compute_params* params,
   NE_ASSERT(src1->type == NE_TYPE_I32);
   NE_ASSERT(ne_nelements(src1) == 5 + bs);  // 5 + bs params
 
-  float freq_base = 10000.0f;
-  memcpy(&freq_base, dst->op_params, sizeof(float));
-  static const float freq_scale = 1.0f;
+  const float freq_base = ((float*)(dst->op_params))[0];
+  const float freq_scale = 1 / ((float*)(dst->op_params))[1];
```

Comment:
Change correspondingly in `ne_compute_forward_rope_f16` and  `kv_cache_init` (`models/model_utils/model_utils.cpp`)

Reply:
Fixed, thanks for your comment